src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Following tags will be not added to spans when the exception is recorder:
```

Comment:
```suggestion
* The following tags are no longer added to spans when an exception is recorded:
```

Reply:
Fixed in 214cad3aba3be7d9bfb5597e3a2df4a8aed63209 with all other entries